### ISSUE TITLE
Is batch_size wrong or forget to specify batch_first True?

### ISSUE THREAD

**Issue by Jia-Bin** [evolvingstuff/SeqMNIST#1]
In lstm_model.py

@ line 10: self.lstm = torch.nn.LSTM(input_scan_dim, hidden_dim)
The default value of batch_first is False. Therefore, we should expect the input x should have
dim like (seq_len, batch_sze, feature_sz)

@line 19: batch_size = x.size(0)
So, this should be x.size(1) for batch_first being False.

Thank you for your clarification.
